public/lib/helpers/__tests__/timeline_helper.js

```diff
@@ -4,6 +4,11 @@ var timelineHelper = require('../timeline_helper')();
 describe('TimelineHelper', function () {
 
   describe('getDescendantPropValue', function () {
+    it('no data', function () {
+      var o = {};
+      expect(timelineHelper.getDescendantPropValue(o, 'p1.p2')).to.be(undefined);
+    });
+
```

Comment:
please add another test for null and empty string 
They both should return respectively null and empty string